### PR TITLE
Enable `global_namespace_import` rule

### PR DESCRIPTION
**Pull request by kenjis**
See https://github.com/codeigniter4/CodeIgniter4/pull/5250

### CODE REVIEW

**Comment by paulbalandan** [comment]
@kenjis please rebase

**Comment by kenjis** [comment]
Rebased.

**Review by MGatner — APPROVED** [review]
I like this. Excited to have it trickle down to projects!

File: src/CodeIgniter4.php [hunk 1]
```diff
@@ -171,7 +171,7 @@ public function __construct()
                 'fix_inline'     => true,
                 'replacements'   => ['inheritDocs' => 'inheritDoc'],
             ],
-            'global_namespace_import'     => false,
+            'global_namespace_import'     => true,
```
Comment (paulbalandan) [inline src/CodeIgniter4.php]:
```suggestion
            'global_namespace_import'     => [
                'import_constants' => false,
                'import_functions' => false,
                'import_classes'   => true,
            ],
```

File: src/CodeIgniter4.php [hunk 2]
```diff
@@ -171,7 +171,11 @@ public function __construct()
                 'fix_inline'     => true,
                 'replacements'   => ['inheritDocs' => 'inheritDoc'],
             ],
-            'global_namespace_import'     => false,
+            'global_namespace_import'     => [
```
Comment (paulbalandan) [inline src/CodeIgniter4.php]:
```suggestion
            'global_namespace_import' => [
```

**Review by paulbalandan — APPROVED** [review]

**Comment by paulbalandan** [comment]
thank you, @kenjis

**Pull request merged**